src/components/grid-base/grid-base.tsx

```diff
@@ -23,6 +23,11 @@ export interface IGridBase {
    */
   loadingState: "loading" | "loaded";
 
+  /**
+   * For infinite scroll, bind it to the next page grid component handler. It will be called every time threshold is reached.
+   */
+  gxInfinite: EventEmitter<void>;
```

Comment:
Maybe we could give this event a more descriptive name? For example, `gxInfiniteThresholdReached`

Reply:
Done!